route-rs-runtime/src/element/drop_element.rs

```diff
@@ -0,0 +1,36 @@
+use crate::element::{AsyncElement, Element};
```

Comment:
For good test coverage, you should also implement an element that only drops some packets. This way we can ensure the case I mentioned above is actually handled.   Maybe a random drop element since that would be useful in real routers to simulate a bad network connection.

Reply:
Thats a good idea! And probably not hard to test if we can set its PRG seed.

Reply:
Created #145 to track that.